src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -257,7 +257,7 @@ const PersonalInfoList = ({ myProf, profile, createSnackbar }) => {
     createSnackbar(status.message, status.type);
   };
 
-  const graduationYear = isAlumni ? (
+  const graduationYear = isAlumni && !profile.PreferredClassYear.trim().isEmpty()? (
```

Comment:
Please make sure you have the `Prettier` VS Code extension enabled to automatically format files when you save them.

In this case, the question mark should be separate from the function call because it is ternary conditional operator and not part of the syntax of the function call.

```suggestion
  const graduationYear = isAlumni && !profile.PreferredClassYear.trim().isEmpty() ? (
```